egs/callhome_diarization/v1/local/VB_resegmentation.py

```diff
@@ -0,0 +1,308 @@
+        line = line.strip('\n')
+        line_split = line.split()
+        uttname_line = line_split[1]
+        if uttname != uttname_line:
+            continue
+        else:
+            utt_rttm_file.write(line + "\n")
+        start_time = int(float(line_split[3]) * 100)
+        duration_time = int(float(line_split[4]) * 100)
+        end_time = start_time + duration_time
+        spkname = line_split[7]
+        if spkname not in speaker_dict.keys():
+            spk_idx = num_spk + 2
+            speaker_dict[spkname] = spk_idx
+            num_spk += 1
+        
+        for i in range(start_time, end_time):
+            if i < 0:
+                raise ValueError(line)
+            elif i >= num_frames:
+                print("{} EXCEED NUM_FRAMES".format(line))
+                break
+            else:
+                if ref[i] == 0:
+                    ref[i] = speaker_dict[spkname] 
+                else:
+                    ref[i] = 1 # The overlapping speech is marked as 1.
+    ref = ref.astype(int)
+
+    print("{} SPEAKERS IN {}".format(num_spk, uttname))
+    print("{} TOTAL, {} SILENCE({:.0f}%), {} OVERLAPPING({:.0f}%)".format(len(ref), np.sum(ref == 0), 100.0 * np.sum(ref == 0) / len(ref), np.sum(ref == 1), 100.0 * np.sum(ref == 1) / len(ref)))
+
+    duration_list = []
+    for i in range(num_spk):
+        duration_list.append(1.0 * np.sum(ref == (i + 2)) / len(ref))
+    duration_list.sort()
+    duration_list = map(lambda x: '{0:.2f}'.format(x), duration_list)
+    print("DISTRIBUTION OF SPEAKER {}".format(" ".join(duration_list)))
+    print("")
+    sys.stdout.flush()
+    utt_rttm_file.close()
+    return ref
+
+def create_rttm_output(uttname, predicted_label, output_dir, channel):
+    num_frames = len(predicted_label)
+
+    start_idx = 0
+    idx_list = []
+
+    last_label = predicted_label[0]
+    for i in range(num_frames):
+        if predicted_label[i] == last_label: # The speaker label remains the same.
+            continue
+        else: # The speaker label is different.
+            if last_label != 0: # Ignore the silence.
+                idx_list.append([start_idx, i, last_label])
+            start_idx = i
+            last_label = predicted_label[i]
+    if last_label != 0:
+        idx_list.append([start_idx, num_frames, last_label])
+
+    with open("{}/{}_predict.rttm".format(output_dir, uttname), 'w') as fh:
+        for i in range(len(idx_list)):
+            start_frame = (idx_list[i])[0]
+            end_frame = (idx_list[i])[1]
+            label = (idx_list[i])[2]
+            duration = end_frame - start_frame
+            fh.write("SPEAKER {} {} {:.2f} {:.2f} <NA> <NA> {} <NA> <NA>\n".format(uttname, channel, start_frame / 100.0, duration / 100.0, label))
+    return 0
+
+def match_DER(string):
+    string_split = string.split('\n')
+    for line in string_split:
+        if "OVERALL SPEAKER DIARIZATION ERROR" in line:
+            return line
+    return 0
+
+def main():
```

Comment:
This needs more documentation, along with a message describing it's usage, and some example arugments.